src/Traveler.js

```diff
@@ -6,8 +6,31 @@ class Traveler {
     this.tripsData = tripsData;
   }
 
-  filterTravelerTrips() {
-    return this.tripsData.filter(trip => trip.userID === this.id);
+  filterTravelerTrips(destinationData) {
+    let userTripsData = this.tripsData.filter(trip => trip.userID === this.id);
+
+    return userTripsData.map(trip => {
+      let destinationInfo = destinationData.find(destinationData => trip.destinationID === destinationData.id);
+
+      return {
+        id: trip.id,
+        travelers: trip.travelers,
+        date: trip.date,
+        duration: trip.duration,
+        status: trip.status,
+        suggestedActivities: trip.suggestedActivities,
+        destination: destinationInfo.destination,
+        totalTripCost: this.calculateTotalTripCost(destinationInfo.estimatedFlightCostPerPerson, trip.travelers, trip.duration, destinationInfo.estimatedLodgingCostPerDay).toFixed(2),
+        agencyFees: (this.calculateTotalTripCost(destinationInfo.estimatedFlightCostPerPerson, trip.travelers, trip.duration, destinationInfo.estimatedLodgingCostPerDay)*.10).toFixed(2),
+        overallCost: (this.calculateTotalTripCost(destinationInfo.estimatedFlightCostPerPerson, trip.travelers, trip.duration, destinationInfo.estimatedLodgingCostPerDay)*1.1).toFixed(2),
```

Comment:
To make this code a little more readable, You could try placing the parameters of these functions into a predefined variable and then just placing the variable in as an argument for each function.

Reply:
@TimNguyen21 Let me know if you want to make any of these updates before I merge it for you.

Reply:
Ok. Make sense.  I does look like a lot of information and it can be assigned to a variable for more readability.  I will make the change push up an updated commit.

Reply:
I made the changes and updated the Pull Request.  If everything else looks good, feel free to merge this pull request.